mllib/src/test/java/org/apache/spark/ml/feature/JavaPCASuite.java

```diff
@@ -107,7 +107,11 @@ public VectorPair call(Tuple2<Vector, Vector> pair) {
       .fit(df);
     List<Row> result = pca.transform(df).select("pca_features", "expected").toJavaRDD().collect();
     for (Row r : result) {
-      Assert.assertEquals(r.get(1), r.get(0));
+      Vector calculatedVector = (Vector)r.get(0);
```

Comment:
Not sure why these don't fail the style checker, but, space after cats, i++ instead of i ++, and 1.0e-8 for clarity